Directory.Build.props

```diff
@@ -5,6 +5,7 @@
     <AssetsDir>$(MSBuildThisFileDirectory)assets\</AssetsDir>
     <VSSDKTargetPlatformRegRootSuffix>RoslynDev</VSSDKTargetPlatformRegRootSuffix>
     <AutoGenerateAssemblyVersion>true</AutoGenerateAssemblyVersion>
+    <NoWarn>$(NoWarn);NU5119</NoWarn>
```

Comment:
>NU5119 [](start = 22, length = 6)

Seems like there should be some property NoDefaultExcludes to avoid this warning  
[https://docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5119](https://docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5119)